docs-2.0/nebula-dashboard/2.deploy-dashboard.md

```diff
@@ -30,7 +30,7 @@ Download the tar package as needed, and it is recommended to select the latest v
 
 | Dashboard package | Nebula Graph version |
 | :----- | :----- |
-| [nebula-graph-dashboard-{{ dashboard.release }}.x86_64.tar.gz](https://oss-cdn.nebula-graph.com.cn/nebula-graph-dashboard/{{ dashboard.release }}/nebula-graph-dashboard-{{ dashboard.release }}.x86_64.tar.gz) | v{{nebula.release}} |
+| [nebula-graph-dashboard-{{ dashboard.release }}.x86_64.tar.gz](https://oss-cdn.nebula-graph.com.cn/nebula-graph-dashboard/{{ dashboard.release }}/nebula-graph-dashboard-{{ dashboard.release }}.x86_64.tar.gz) | 2.5.0 |
```

Comment:
Remember to update both version numbers when the new doc version is released.

Reply:
why mv macro to hardcode